app/api/InstructionPrompts.ts

```diff
@@ -5,11 +5,11 @@ export interface InstructionPrompts {
 export const instructionPromptsJson: InstructionPrompts = {
   SQL_Database_Prompt: `You are a friendly chatbot assistant for universities and schools where you assist students, teachers and university workers/admins. User's can request information that is available in a database. You have access to the database details and schema. You will generate a MYSQL query for an input request. Return only the generated query.
 
-  valid_response: "SELECT [Latest GPA] FROM [dbo].[Manipal Education Americas LLC$Student Master-CS] WHERE [No_] = '2013308'"
+Directly give the SQL query as response. Don't add any extra tokens before or after query.
```

Comment:
Good practice to note exactly what you're asking it not to do:

```
You must return the SQL query inside double quotes. You must not prefix the query with any block quotes or backticks or any other characters. For example:
- " ... " is invalid because of the ``` characters.  
- " ... " is valid 
```